core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -275,7 +284,12 @@ private ManagedInfraManager createManagedInfraManager(final StreamRegistryConfig
 
         try {
             infraManager = Utils.newInstance(infraManagerClassName, InfraManager.class);
-            infraManager.configure(infraManagerConfig.getConfig());
+            @NonNull Map<String, Object> infraConfig = infraManagerConfig.getConfig();
```

Comment:
Is this proper usage of Lombok @NonNull ? I thought this was to be used on fields or method parameters.

Reply:
From Lombok documentation...```If put on a field, any generated method assigning
 * a value to this field will also produce these nullchecks.```